Parsers/ASimWebSession/Test/Cisco_Firepower_vimWebSession_SchemaTest.csv

```diff
@@ -0,0 +1,115 @@
+﻿Result
+"(1) Warning: Missing recommended field [DstHostname]"
+"(1) Warning: Missing recommended field [EventResultDetails]"
```

Comment:
missing?
![image](https://github.com/Azure/Azure-Sentinel/assets/97222872/52e4716e-40dc-44b8-83e7-8df238712ebb)

Reply:
Not available